docs/remove_controller.md

```diff
@@ -0,0 +1,32 @@
+# Remove or replace a controller
+
+You can manually remove or replace a controller from a multi-node k0s cluster (>=3 controllers) without downtime.
+
+## Remove a controller
+
+You first have to delete the controller from Kubernetes itself.
+To do so, run the following commands from the controller:
+
+```shell
+k0s kubectl drain --ignore-daemonsets --delete-emptydir-data <controller>
+k0s kubectl delete node <controller>
+```
+
+Then you need to remove it from the etcd cluster.
+
+```shell
+PEER_ADRESS=$(k0s etcd member-list | python -m json.tool | grep <controller> | cut -d"\"" -f4)
+k0s etcd leave --peer-address $PEER_ADRESS
+```
```

Comment:
Maybe it would be better to explain, with example command outputs, how to find the correct peer address instead of scripting it fully. Yes, a sample script is useful too but I fear that it "hides" some details from not so experienced folks.